solr/solrj-jetty/src/java/org/apache/solr/client/solrj/impl/PreemptiveBasicAuthClientBuilderFactory.java

```diff
@@ -66,15 +69,16 @@ public static void setDefaultSolrParams(SolrParams params) {
   }
 
   @Override
-  public void close() throws IOException {}
-
-  @Override
-  public void setup(Http2SolrClient client) {
+  public void setup(SolrClient client) {
+    if (client instanceof Http2SolrClient == false) {
+      return;
```

Comment:
[-1] Perhaps I'm misreading this, but it seems trappy to return quietly when the user passes in a client that we don't support configuring.  Shouldn't we throw an exception or something here instead?